src/components/Cart.svelte

```diff
@@ -0,0 +1,94 @@
+<script>
+    import {totalPrice, cart} from "../store/cart"
+    import {products} from '../store/products.ts';
+    import {
+        Button,
+        Modal,
+        ModalHeader,
+        ModalBody,
+        ModalFooter,
+        Table,
+        Input
+    } from 'sveltestrap';
+
+    let isOpen = false;
+    const toggle = () => (isOpen = !isOpen);
+
+    const deleteAll = () => {
+        cart.set([])
+    }
+
+    const checkout = () => {
+        alert(JSON.stringify($cart))
+        deleteAll()
+        toggle()
+    }
+
+    const deleteProduct = (productId) => {
+        $cart.map((item, index) => {
+            if (item.id === productId) {
+                $cart.splice(index, 1)
+            }
+            $cart = $cart
+        })
+    }
+
+    $: $cart.map((item, index) => {
+        if (item.quantity === 0) {
+            $cart.splice(index, 1)
+        }
+    })
+</script>
+
+<Button on:click={toggle} class="bg-success text-dark">
+    {$totalPrice} Kč
+</Button>
+
+<Modal isOpen={isOpen} {toggle} scrollable>
+    <ModalHeader {toggle}>
+        Cart
+    </ModalHeader>
+    <ModalBody>
+        {#if $cart.length === 0}
+            <p>
+                Cart is empty. Add items to cart.
+            </p>
+        {:else}
+            <Table responsive>
+                <thead>
+                <tr>
+                    <th></th>
+                    <th>Název</th>
+                    <th>Počet</th>
+                    <th>Cena</th>
+                </tr>
+                </thead>
+                {#each $cart as item (item.id)}
+                    <tbody>
+                    <tr>
+                        <th scope="row">
+                            <Button on:click={() => deleteProduct(item.id)}>
+                                &times;
+                            </Button>
+                        </th>
+                        <td>
+                            {$products[item.id].name}
```

Comment:
if we would store the whole product object in the cart store, then we don't need the products store here